fairscale/optim/oss.py

```diff
@@ -140,6 +140,11 @@ def partition_parameters(self) -> List[List[dict]]:
                     param_group_rank["params"] = params
                     self._partition_parameters[rank].append(param_group_rank)
 
+            assert min(sum(len(pg["params"]) for pg in partition) for partition in self._partition_parameters) > 0, (
+                "One or more empty shards detected, the world size is too big or the model too small."
```

Comment:
How about printing out the size of the model and world size for easy debugging?

Reply:
sure, I'm guessing that typically people know the world size though since they're the ones who started the job, but I can do that.I'm also guessing that people would not change models just to use OSS, but I can update the message with both informations still